docusaurus/docs/learn/core-concepts/services/overview.mdx

```diff
@@ -143,3 +143,339 @@ This strategy drives costs in the same way as the `smartrouting` strategy. Howev
 ##### `random`
 This strategy does not change terminator weights. It does simple random selection across all terminators of the highest precedence. 
 
+## Practical Service Hosting
+
+### Edge Router Tunneler Hosting
+
+#### Single Application Endpoint
+When hosting services with the edge router tunneler (ER/T) combination you'll need to use a service configurations. We're going
```

Comment:
I'd like to avoid "we/we're/I" type language. This kind of doc should simply be factual and use statements and not personal.

Reply:
fixed all the we/I/you language